schema_based_on_diagram.sql

```diff
@@ -0,0 +1,52 @@
+CREATE TABLE patients{
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(50),
+    date_of_birth DATE NOT NULL
+}
+
+CREATE TABLE medical_histories{
+    id SERIAL PRIMARY KEY,
+     admitted_at TIMESTAMP NOT NULL,
+    patient_id INT NOT NULL,
+    status VARCHAR(50),
+    CONSTRAINT patients_fk FOREIGN KEY (patient_id) REFERENCES patients(id)
+}
+
+CREATE TABLE invoices{
+    id SERIAL PRIMARY KEY,
+    total_amount DECIMAL NOT NULL,
+    generated_at TIMESTAMP NOT NULL,
+    payed_at TIMESTAMP NOT NULL,
+    medical_history_id SERIAL NOT NULL,
+    CONSTRAINT history_fk FOREIGN KEY (medical_history_id) REFERENCES medical_histories(id)
```

Comment:
- [ ] We know that the `SERIAL` value type is an alias for `BIGINT UNSIGNED NOT NULL AUTO_INCREMENT UNIQUE` however, let's not use it for the reference column `medical_history_id` instead, we want the reference column to be added when creating a new row and should not be auto-incremented. Let's replace `medical_history_id SERIAL NOT NULL,` with `medical_history_id INT NOT NULL,`